ci/build_container/build_recipes/boringssl.sh

```diff
@@ -1,10 +1,16 @@
+#!/bin/bash
+
 set -e
 
+COMMIT=be2ee342d3781ddb954f91f8a7e660c6f59e87e5
+
 git clone https://boringssl.googlesource.com/boringssl
 cd boringssl
-git reset --hard be2ee342d3781ddb954f91f8a7e660c6f59e87e5
+git reset --hard $COMMIT
 cmake .
 make
 cp -r include/* $THIRDPARTY_BUILD/include
 cp ssl/libssl.a $THIRDPARTY_BUILD/lib
 cp crypto/libcrypto.a $THIRDPARTY_BUILD/lib
+cd ..
+rm -rf boringssl
```

Comment:
It makes scripts self-contained, but I indeed missed that cleanup in Makefile, so I'm going to remove it from scripts (it seems that you prefer it this way and I don't care too much).